blazor/gantt-chart/rows.md

```diff
@@ -548,6 +548,115 @@ The following code example shows how to drag and drop a row on button click acti
 }
 ```
 
+### Dynamically hide rowdragdrop icons
+
+By default, it renders all records with a rowdragdrop icon. In this guide, we'll show you how to dynamically conditionally hide the rowdragdrop icon based on specific conditions using the [`RowDataBound`](https://blazor.syncfusion.com/documentation/gantt-chart/events#rowdatabound) event.
```

Comment:
dynamically conditionally not proper sentence. Dont mention it as as in this guide